sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubProducer.java

```diff
@@ -77,17 +78,32 @@
  *
  * {@codesnippet com.azure.messaging.eventhubs.eventhubproducer.send#publisher-sendOptions}
  *
+ * <p><strong>Publish events using an {@link EventDataBatch}.</strong></p>
+ *
+ * Developers can create {@link EventDataBatch batches}, add the events they want in that batch, and publish these
+ * events together. When creating a {@link EventDataBatch batch}, developers can specify a set of {@link BatchOptions
+ * options} to configure this batch.
+ *
+ * In the scenario below, the developer is creating a networked video game. They want to receive telemetry about their
```

Comment:
Wait, what?
We are adding sample scenarios in code?

Reply:
This scenario text is in service of describing the code snippet that follows. There is no harm in this so long as it does not become a wall of text, as the focus should always be on the code sample itself.